src/pocketmine/Player.php

```diff
@@ -3241,7 +3241,14 @@ public function handleItemFrameDropItem(ItemFrameDropItemPacket $packet) : bool{
 
 		$tile = $this->level->getTile($this->temporalVector->setComponents($packet->x, $packet->y, $packet->z));
 		if($tile instanceof ItemFrame){
+			$ev = new PlayerInteractEvent($this, $this->inventory->getItemInHand(), $tile->getBlock(), $tile->getBlock()->getDamage(), PlayerInteractEvent::LEFT_CLICK_BLOCK);
```

Comment:
I don't think the the damage value as clicked-face here is correct. See https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/block/ItemFrame.php#L111-L116 for placement values.

Other than that, it works fine as a resolution for this problem.